src/templates/finops-hub/modules/dataFactory.bicep

```diff
@@ -4593,6 +4596,7 @@ resource pipeline_ExecuteIngestionETL 'Microsoft.DataFactory/factories/pipelines
         ]
         userProperties: []
         typeProperties: {
+          batchCount: dataExplorerIngestionCapacity // Concurrency limit
```

Comment:
@bwatts64 Do we need to batch these or can we just call .ingest on all partitions at once?